src/OnlineRetailPortal.Contracts/Contracts/ICategoryService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OnlineRetailPortal.Contracts
+{
+    public interface ICategoryService
+    {
+        Task<GetCategoriesServiceResponse> GetCategoriesAsync();
```

Comment:
Rename GetCategoriesServiceResponse to CategoriesServiceResponse

Reply:
should i also change GetCategoriesServiceResponseTranslator  to  CategoriesServiceResponseTranslator  in Translator folder